CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ SET(HEADERS
   ${CSD}/include/roboptim/trajectory/visualization/limit-speed.hh
   ${CSD}/include/roboptim/trajectory/visualization/speed.hh
   ${CSD}/include/roboptim/trajectory/visualization/trajectory.hh
+  ${CSD}/include/roboptim/trajectory/visualization/matplotlib.hh
```

Comment:
`include/roboptim/trajectory/visualization/matplotlib/cubic-b-spline.hh` may be better.